Sources/Sentry/include/SentryDataCategory.h

```diff
@@ -17,5 +17,6 @@ typedef NS_ENUM(NSUInteger, SentryDataCategory) {
     kSentryDataCategoryMetricBucket = 8,
     kSentryDataCategoryReplay = 9,
     kSentryDataCategoryProfileChunk = 10,
-    kSentryDataCategoryUnknown = 11
+    kSentryDataCategoryUnknown = 11,
+    kSentryDataCategorySpan = 12,
```

Comment:
```suggestion
   kSentryDataCategorySpan = 11,
   kSentryDataCategoryUnknown = 12,
```

Reply:
k, so it does not matter if the raw value changes for unknown